.github/workflows/start_benchmarks.yml

```diff
@@ -4,11 +4,12 @@ name: Start all benchmarks
 on:
   push:
     branches:
-      - 'main'
+      - "main"
   workflow_dispatch:
 
 jobs:
   start-benchmarks:
+    if: ${{ (github.event_name == 'push' && github.repository == 'zama-ai/tfhe-rs') ||  github.event_name == 'workflow_dispatch' }}
```

Comment:
I don't quite get it. Does this mean that push on other repo (forked ones ?) can trigger the CI ?

Reply:
no it's to avoid double launching with internal

Reply:
Ok makes sense now.